ms-common/src/main/java/net/trajano/ms/core/JwtClaimsSetPrincipal.java

```diff
@@ -1,9 +1,15 @@
 package net.trajano.ms.core;
 
+import org.jose4j.jwt.JwtClaims;
+import org.jose4j.jwt.MalformedClaimException;
+
 import java.net.URI;
 import java.security.Principal;
-
-import com.nimbusds.jwt.JWTClaimsSet;
+import java.util.Collections;
+import java.util.Map;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Map'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)